### ISSUE TITLE
Possible error in the documentation

### ISSUE THREAD

**Issue by Introvertigo** [uspgamedev/luasteam#30]
The for the sendMessageToConnection function is:

> local result = Steam.networkingSockets.sendMessageToConnection(connection, "Ping", Steam.networkingSockets.flags.Reliable)

Shouldnt the flag key be **Send_Reliable** instead of just **Reliable** ?

https://luasteam.readthedocs.io/en/stable/networking_sockets.html#networkingSockets.sendMessageToConnection

**Comment by zorfmorf** [comment]
Yes, the example seems wrong. I will get it fixed. I think in this case it will still default to reliable anyway though.
